presto-native-execution/src/test/resources/external_functions.json

```diff
@@ -85,6 +85,82 @@
           "isOrderSensitive": false
         }
       }
+    ],
+    "avg_partial": [
+      {
+        "docString": "Partial companion function of avg, returning accumulator structs {sum, count} of input integers",
+        "functionKind": "AGGREGATE",
+        "outputType": "ROW(DOUBLE, BIGINT)",
+        "paramTypes": [
+          "INTEGER"
+        ],
+        "schema": "test_schema",
+        "routineCharacteristics": {
+          "language": "CPP",
+          "determinism": "DETERMINISTIC",
+          "nullCallClause": "CALLED_ON_NULL_INPUT"
+        },
+        "aggregateMetadata": {
+          "intermediateType": "ROW(DOUBLE, BIGINT)",
+          "isOrderSensitive": false
+        }
+      }
+    ],
+    "avg_merge": [
```

Comment:
The json entries LGTM; 
nit: it may be beneficial to use a more specific naming; for e.g. instead of `avg_merge`, use `avg_merge_only` / `avg_merge_intermediate` to indicate this is not a final step (AOT regular agg functions where there's only one reduction / merge step). one can definitely infer this from the test cases; so, please ignore if you think the current naming is clear.

Reply:
Hi @Ali-P, the companion function names (e.g., avg_merge) are auto-generated and follow a name convention (https://github.com/facebookincubator/velox/discussions/4493). E.g, for avg(), its companion functions are avg_partial, avg_merge, etc. For another aggregation function sum(), its companion functions are sum_partial, sum_merge, etc. So I cannot just modify the name of avg's companion functions.

Reply:
SGTM :)